src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
-body {
-  margin: 0;
+@import "../styles/reset.scss";
+@import "../styles/variables.scss";
+
+.calendar-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-day-gap;
+  width: (($calendar-day-size + $calendar-day-gap + $calendar-day-border) * 7);
```

Comment:
Nice naming 👍 You use `box-sizing: border-box` , variable `$calendar-day-border` is unnecessary here.

Reply:
Ok, thnx)